### PR TITLE
docs: update `Add SSL Certificate` article

### PR DESCRIPTION
**Pull request by CarolFP1896**
## Description of changes
With the changes in the SSL Certificate flow in the web-app, it was necessary to update the `Add an SSL Certificate` article in order to be consistent with the current resource flow and make it easier and more intuitive for the user to follow.   

## GitHub issues resolved by this PR

- [x] closes [DVN-14187]

## Quality Assurance

- Once the changes in this PR are merged and deployed, success criteria is: having the flow of creating an SSL Certificate documented in a clear and simple step-by-step, which guides the user to their objective (creating an SSL Certificate).

## More info

N/A

[DVN-14187]: https://devopness.atlassian.net/browse/DVN-14187?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/ssl-certificates/add-ssl-certificate.md [hunk 1]
```diff
@@ -10,29 +10,14 @@ links:
...
-1. Choose the certificate authority and click `CONFIRM`
-2. Wait for the `ssl-certificate:deploy` action to be completed
+1. On Devopness, navigate to a project then select an environment
+2. Find the `SSL Certificates` card
+3. Click `View` in the `SSL Certificates` card to see a list of existing `SSL Certificates`
+4. On the upper-right corner of the list, click `ADD SSL CERTIFICATE`
+5. On the Virtual host dropdown menu, choose a Virtual Host.
+   > The name of your SSL certificate will be the name of the chosen Virtual Host.
+6. Choose the certificate authority and click `CONFIRM`
+7. An action to install the SSL Certificate will start
```
Comment (jfoliveira) [inline docs/ssl-certificates/add-ssl-certificate.md:23]: Perhaps clarify who will start the action?

```suggestion
7. An action will be started by Devopness to issue an SSL Certificate and install it for the chosen Virtual Host
```

File: docs/ssl-certificates/add-ssl-certificate.md [hunk 2]
```diff
@@ -10,29 +10,14 @@ links:
...
-1. On the chosen Devopness environment, click `View` in the `Applications` card to see a list of existing `Applications`
-1. In the list of applications, find the application you want to add an SSL certificate to and click the `NAME` of the application
-1. On the upper-right corner click `SSL`
-1. Choose the certificate authority and click `CONFIRM`
-2. Wait for the `ssl-certificate:deploy` action to be completed
+1. On Devopness, navigate to a project then select an environment
+2. Find the `SSL Certificates` card
+3. Click `View` in the `SSL Certificates` card to see a list of existing `SSL Certificates`
+4. On the upper-right corner of the list, click `ADD SSL CERTIFICATE`
+5. On the Virtual host dropdown menu, choose a Virtual Host.
```
Comment (jfoliveira) [inline docs/ssl-certificates/add-ssl-certificate.md:20]: Whenever an UI element name is mentioned in our docs, please make sure to highlight it `like here` and use the exact upper case or lower case letters used for that element in the UI.
That way users can quickly relate to the UI elements when using our docs.

Please double check, cause I believe we spell `Virtual Host` capitalizing the first letters, but I might be mistaken.

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/ssl-certificates/add-ssl-certificate.md (hunk 2 above)
Comment (CarolFP1896) [reply docs/ssl-certificates/add-ssl-certificate.md]: Done!

Comment (jfoliveira) [reply docs/ssl-certificates/add-ssl-certificate.md]: @CarolFP1896 
"choose a `Virtual Host` doesn't need to be high" is not referring to an UI element and therefore should not be highlighted.
I see you adopted this pattern to other names in the docs.
Please, as mentioned above, just highlight the "UI elements" and not all texts that mention the name of a resource type.

**Review by jfoliveira — DISMISSED** [review]

**Comment by CarolFP1896** [comment]
I have read the CLA document and I hereby sign the CLA

**Comment by jfoliveira** [comment]
recheck

**Comment by thlmenezes** [comment]
recheck

File: docs/ssl-certificates/add-ssl-certificate.md [hunk 3]
```diff
@@ -10,29 +10,14 @@ links:
...
-1. In the list of applications, find the application you want to add an SSL certificate to and click the `NAME` of the application
-1. On the upper-right corner click `SSL`
-1. Choose the certificate authority and click `CONFIRM`
-2. Wait for the `ssl-certificate:deploy` action to be completed
+1. On Devopness, navigate to a project then select an environment
+2. Find the `SSL Certificates` card`
+3. Click `View` in the `SSL Certificates` card to see a list of existing `SSL Certificates`
+4. On the upper-right corner of the list, click `ADD SSL CERTIFICATE`
+5. On the `Virtual Host` dropdown menu, choose a virtual host
+   > The name of your SSL certificate will be the name of the chosen virtual host
```
Comment (thlmenezes) [inline docs/ssl-certificates/add-ssl-certificate.md:21]: @CarolFP1896 although they have the same visual effect, semantically, in Markdown, it is preferred to use unordered list notation; and in our docs, it is more common the unordered list approach [1].

```suggestion
   - The name of your SSL certificate will be the name of the chosen virtual host
```

[1]

```bash
$ grep -re '    > [A-Z]' `find docs -type f` | wc -l
44
$ grep -re '    - [A-Z]' `find docs -type f` | wc -l
85
```

File: docs/ssl-certificates/add-ssl-certificate.md [hunk 4]
```diff
@@ -10,29 +10,14 @@ links:
...
-1. Access your DNS service management panel
-    > If you don't have your own domains configured with a DNS service, you can use free services such as [FreeDNS](https://freedns.afraid.org/) or hire paid managed DNS services such as [AWS Route53](https://aws.amazon.com/route53/), [Cloudflare DNS](https://www.cloudflare.com/dns/), or any [notable managed DNS service provider](https://en.wikipedia.org/wiki/List_of_managed_DNS_providers) that suits your needs.
-1. Configure a DNS record for your application domain
-    > Follow the specific instructions on your chosen DNS service documentation
-1. Verify your DNS record is pointing to your Devopness managed server
-  > This can be verified using a DNS propagation checker such as [whatsmydns](https://www.whatsmydns.net/) or [DNS Checker](https://dnschecker.org/)
-
-</details>
-
-Once you have your application DNS record correctly pointing to the IP Address of your Devopness managed server, please follow the instructions below to issue a SSL Certificate to your application:
```
Comment (thlmenezes) [inline docs/ssl-certificates/add-ssl-certificate.md:32]:
```suggestion
Once you have your virtual host DNS record correctly pointing to the IP Address of your Devopness managed server, please follow the instructions below to issue a SSL Certificate to your virtual host:
```

**Review by thlmenezes — CHANGES_REQUESTED** [review]

File: docs/ssl-certificates/add-ssl-certificate.md [hunk 5]
```diff
@@ -10,29 +10,14 @@ links:
...
-
-1. On the chosen Devopness environment, click `View` in the `Applications` card to see a list of existing `Applications`
-1. In the list of applications, find the application you want to add an SSL certificate to and click the `NAME` of the application
-1. On the upper-right corner click `SSL`
-1. Choose the certificate authority and click `CONFIRM`
-2. Wait for the `ssl-certificate:deploy` action to be completed
+1. On Devopness, navigate to a project then select an environment
+2. Find the `SSL Certificates` card`
+3. Click `View` in the `SSL Certificates` card to see a list of existing `SSL Certificates`
```
[the remark this answers is not in the document]
Comment (CarolFP1896) [reply docs/ssl-certificates/add-ssl-certificate.md]: Done!

File: docs/ssl-certificates/add-ssl-certificate.md (hunk 3 above)
Comment (CarolFP1896) [reply docs/ssl-certificates/add-ssl-certificate.md]: Done!

File: docs/ssl-certificates/add-ssl-certificate.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (thlmenezes) [reply docs/ssl-certificates/add-ssl-certificate.md:32]: @CarolFP1896 the issue has not been prioritized or moved to any sprint, so between the merge of this PR and the PR that resolves that issue the users and Devopness team will no longer have any written resource to guide users that a DNS record should be created and how to do it.

Even with that task in Jira, I still believe that we should revert this change.

On the occasion that the task is being resolved, the PR can create the new article and delete these steps.

File: docs/ssl-certificates/add-ssl-certificate.md [hunk 6]
```diff
@@ -10,29 +10,14 @@ links:
...
-1. In the list of applications, find the application you want to add an SSL certificate to and click the `NAME` of the application
-1. On the upper-right corner click `SSL`
-1. Choose the certificate authority and click `CONFIRM`
-2. Wait for the `ssl-certificate:deploy` action to be completed
+1. On Devopness, navigate to a project then select an environment
+1. Find the `SSL Certificates` card
+1. Click `View` in the `SSL Certificates` card to see a list of existing `SSL Certificates`
+1. On the upper-right corner of the list, click `ADD SSL CERTIFICATE`
+1. On the `Virtual Host` dropdown menu, choose a virtual host
+   - The name of your SSL certificate will be the name of the chosen virtual host
```
Comment (thlmenezes) [inline docs/ssl-certificates/add-ssl-certificate.md:21]:
```suggestion
    - The name of your SSL certificate will be the name of the chosen virtual host
```

File: docs/ssl-certificates/add-ssl-certificate.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (CarolFP1896) [reply docs/ssl-certificates/add-ssl-certificate.md:32]: Ok, so let's revert it until the task is done, then we change it.

File: docs/ssl-certificates/add-ssl-certificate.md [hunk 7]
```diff
@@ -10,29 +10,31 @@ links:
...
+Before an SSL certificate can be issued, a valid DNS record must point to the server on which the virtual host is deployed, so the SSL Certificate can be issued and validated for the configured virtual host web domain. For example, if your virtual host is publicly accessible through the domain `my-application.example.com`, the subdomain `my-application` on the domain `example.com` must be configured to point to your server IP address.
 
 <details open>
   <summary>Steps to configure a DNS record pointing to your servers:</summary>
 
-1. Copy the IP Address of the server where the application is running
-    > Follow the {% mentionPost "/docs/servers/find-server-ip-address" %} guide for detailed instructions
+1. Copy the IP Address of the server where the virtual host is running
+    - Follow the {% mentionPost "/docs/servers/find-server-ip-address" %} guide for detailed instructions
```
Comment (thlmenezes) [inline docs/ssl-certificates/add-ssl-certificate.md:22]:
```suggestion
    - Follow the guide {% mentionPost "/docs/servers/find-server-ip-address" %}
```

**Review by thlmenezes — CHANGES_REQUESTED** [review]

**Review by thlmenezes — APPROVED** [review]

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**